src/main/java/org/arkecosystem/crypto/transactions/builder/AbstractTransaction.java

```diff
@@ -12,13 +13,18 @@
     public AbstractTransaction() {
         this.transaction = new Transaction();
         this.transaction.type = this.getType();
-        this.transaction.fee = Fee.get(this.getType());
+        this.transaction.fee = Fee.getCoreFee(this.getType());
         this.transaction.timestamp = Slot.time();
-        this.transaction.version = 1;
+        this.transaction.version = 2;
         this.transaction.network = Network.get().version();
+        this.transaction.typeGroup = TransactionTypeGroup.CORE;
+        this.transaction.nonce = 0;
     }
 
     public AbstractTransaction sign(String passphrase) {
+        if (this.transaction.type == TransactionType.MULTI_SIGNATURE_REGISTRATION && this.transaction.version == 2){
+             throw new UnsupportedOperationException("Version 2 MultiSignatureRegistration is not supported in java sdk");
```

Comment:
Why not?

Reply:
me and @supaiku0 didn't find well representative library for java which supports schnorr signature